ui/src/track.ts

```diff
@@ -1,4 +0,0 @@
-export enum Event {
```

Comment:
Yeah, we were only using this type in `FollowToggle` and we couldn't use the types at the call-site, because of special svelte syntax for the handlers like `on:eventname`. `on:Event.Track` doesn't compile...

So I deleted this file:

<img width="725" alt="Screenshot 2020-10-27 at 15 24 31" src="https://user-images.githubusercontent.com/158411/97315256-d1c5d200-1868-11eb-9374-54d560f7329d.png">